src/Microsoft.AspNetCore.Session/DistributedSession.cs

```diff
@@ -178,6 +177,7 @@ private void Load()
                     }
                     else if (!_isNewSessionKey)
                     {
+                        _store = new Dictionary<EncodedKey, byte[]>();
```

Comment:
This looks good, but will it break anything else I wonder.

Reply:
Yep, it breaks 😞